<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="9" column="0">
       <widget class="QLabel" name="lb_alumbrado">
        <property name="text">
         <string>Alumbrado</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_inicio"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lb_consumo_reactiva">
        <property name="text">
         <string>Consumo Reactiva</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lb_matricula">
        <property name="text">
         <string>Matricula</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="le_valor_pagar"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lb_valor_pagar">
        <property name="text">
         <string>Valor a Pagar</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_matricula"/>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="le_kw"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lb_kw">
        <property name="text">
         <string>Kw/h</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_inicio">
        <property name="text">
         <string>Fecha Inicio</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_final"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lb_consumo_activa">
        <property name="text">
         <string>Consumo Activa</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="le_consumo_reactiva"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lb_valor_kw">
        <property name="text">
         <string>Valor Kw</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="le_contribucion"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lb_final">
        <property name="text">
         <string>Fecha Final</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="le_consumo_activa"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lb_contribucion">
        <property name="text">
         <string>Contribucion</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="le_valor_kw"/>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="le_alumbrado"/>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QPushButton" name="btn_guardar">
        <property name="text">
         <string>Guardar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>342</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
